src/main/scala/verifier/DefaultMasterVerifier.scala

```diff
@@ -262,25 +261,30 @@ class DefaultMasterVerifier(config: Config, override val reporter: Reporter)
      ++ methodVerificationResults)
   }
 
-  private def createInitialState(member: ast.Member, program: ast.Program): State = {
+  private def createInitialState(member: ast.Member, program: ast.Program,
+                                 functionData: Map[ast.Function, FunctionData], predicateData: Map[ast.Predicate, PredicateData]): State = {
     val quantifiedFields = InsertionOrderedSet(ast.utility.QuantifiedPermissions.quantifiedFields(member, program))
     val quantifiedPredicates = InsertionOrderedSet(ast.utility.QuantifiedPermissions.quantifiedPredicates(member, program))
     val quantifiedMagicWands = InsertionOrderedSet(ast.utility.QuantifiedPermissions.quantifiedMagicWands(member, program)).map(MagicWandIdentifier(_, program))
 
-    State(qpFields = quantifiedFields,
+    State(program=program, functionData = functionData,
```

Comment:
For consistency: please break into two lines, and format as `k = v` (space around `=`)